frontend/widgets/SwellWelcome/components/LoggedInView/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { I18n } from '@shopgate/engage/components';
+import Link from '@shopgate/pwa-common/components/Link';
+import { CHECKOUT_PATH } from '@shopgate/pwa-common/constants/RoutePaths';
+import getSwellSdk from '../../../../helpers/getSwellSdk';
+import styles from './style';
+
+/**
+ * @returns {string}
+ */
+const LoggedInView = ({
+  buttonBackgroundColor,
+  buttonFontColor,
+  firstName,
+  headerTextColor,
+  historyButtonText,
+  points,
+  redeemButtonText,
+}) => {
+  useEffect(() => {
+    if (!window.swellAPI) {
+      getSwellSdk();
+    }
+  });
+  /**
+   * @param {Class} e event Class passed
+   */
+  const showModal = (e) => {
+    e.preventDefault();
+    window.swellAPI.showPopupByType('HistoryPopup');
+  };
+
+  return (
+    <div className={styles.container}>
+      <I18n.Text className={styles.heading(headerTextColor)} string="swell.welcome.greeting" params={{ firstName }} />
+      <I18n.Text className={styles.heading(headerTextColor)} string="swell.welcome.points" params={{ points }} />
```

Comment:
I'm not sure they should be one translation. Not sure how to style the locale string since new line character is ignored when I tested implementing this.